include/tennicam_client/transform.hpp

```diff
@@ -1,38 +0,0 @@
-#pragma once
-
-#include <armadillo>  // vec
-#include <array>
-#include "shared_memory/serializer.hpp"
-#include "shared_memory/shared_memory.hpp"
-
-namespace tennicam_client
-{
-/**
- * @brief represents a 3d transform (translation and rotation)
- */
-class Transform
-{
-public:
-    /**
-     * rotation: array of 3 angles in radians
```

Comment:
This could be a bit more specific, since there are different ways of representing a rotation with three angles.  I assume they are XYZ Euler angles?